pkg/postinstall

```diff
@@ -63,35 +63,65 @@ if [ "${NETWORK}" != 'yes' ]; then
 fi
 
 # Debian based distros
-if [ -d /etc/network ]; then
-    # Prepare network files
-    cp /etc/network/interfaces /etc/network/interfaces.$(date "+%s")
-    rm -rf /etc/network/interfaces.d
-    rm -rf /etc/netplan/*
-
-    if [ "${NETWORK_EC2}" = 'yes' ]; then
-        cat > /etc/network/interfaces <<EOT
+netplan=$(dpkg-query -W -f='${Status}' netplan.io | awk '{print $3}')
```

Comment:
Probably I would limit the changes in post-install script to minimum:
- not necessary to detect netplan.io package
- leave EC2 on `/etc/network/interfaces`
- non-EC2:
   - always drop `/etc/netplan/*`
   - don't generate any Netplan stub (empty config with network/version/renderer)